script/Arcueid/Day 7/07_00_ARC07_3F.txt

```diff
@@ -1067,28 +1197,35 @@ The moment I uttered those words, I lost every right I had to lie to myself.
 [sha:b78ec5e1d74e31ba83a108e1214c1b295cc7ab95]{
 -- Page 419, Offset 17621.
 -- 「早く。わたし、今の台詞もう一度聞きたい」
-"Come on. I want to hear that line just now one more time."
+"Come on. I want to hear that line just one more time."
 }
 [sha:1cf05ed018a7b49dce0de4a37b01322bd4e4b3cb]{
 -- Page 420, Offset 17622.
 -- 囁くように要求してくるアルクェイド。
-Arcueid makes a request to me like a whisper.
+// 要求 has a bit of strength to it, but 'demand' is too much
+Arcueid continues to entreat me, her voice almost a whisper.
```

Comment:
maybe supplicate? entreat is a little archaic imo

Reply:
Hmm, supplicate reads a little _too_ grovel-y to me. Implore?

Reply:
Wheedle or cajole perhaps? Or rearrange to something like
> Arcueid's coaxing voice is little more than a whisper.

Reply:
That last one works well